pkgs/applications/networking/twingate/default.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
   version = "1.0.12";
```

Comment:
Shouldn't we upgrade to the latest version?

Reply:
We haven't tested headless on it at all. It's 100% working version on nixos, wanted to init with it